src/sources/soundsourceffmpeg.cpp

```diff
@@ -88,21 +203,15 @@ SoundSourceFFmpeg::~SoundSourceFFmpeg() {
 }
 
 SoundSource::OpenResult SoundSourceFFmpeg::tryOpen(const AudioSourceConfig& /*audioSrcCfg*/) {
-    AVDictionary *l_iFormatOpts = nullptr;
-
-    const QString localFileName(getLocalFileName());
-    qDebug() << "New SoundSourceFFmpeg :" << localFileName;
```

Comment:
A general warning message with the actual filenname will be printed by 
SoundSourceProxy shortly after.

Currently each subclass does its own logging, there is no common concept. 
Another open issue that's on my list ;)

On 08.09.2016 20:53, Daniel Schürmann wrote:
>
> In src/sources/soundsourceffmpeg.cpp 
> <https://github.com/mixxxdj/mixxx/pull/1003#discussion_r78067376>:
>
> > @@ -88,21 +203,15 @@ SoundSourceFFmpeg::~SoundSourceFFmpeg() {
> >  }
> >
> >  SoundSource::OpenResult SoundSourceFFmpeg::tryOpen(const AudioSourceConfig& /*audioSrcCfg*/) {
> > -    AVDictionary *l_iFormatOpts = nullptr;
> > -
> > -    const QString localFileName(getLocalFileName());
> > -    qDebug() << "New SoundSourceFFmpeg :" << localFileName;
>
> Do we still print the file name somewhere?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/mixxxdj/mixxx/pull/1003/files/14399cf62b5d5ee96a4d0d3a361af8ab165b2144#r78067376>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ABZ-aixHW_N2cfudAW-JbTCTGPf-461jks5qoFmsgaJpZM4J2QNu>.
>